resources/assets/components/WithReviewing/index.js

```diff
@@ -196,6 +196,13 @@ const reviewComponent = (Component, data) => {
             // Remove the deleted post from the state
             delete(newState.posts[postId]);
 
+            // Remove the postId from the state.
+            const postIdIndex = newState.postIds.indexOf(postId);
```

Comment:
We could use `newState.postIds.includes(postId)` for a lil' better readability. 🔢

Reply:
@DFurnes I think `includes()` just returns a boolean and I was using the index to actually remove the value from the array.